packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -234,6 +252,23 @@ abstract class Action<T extends Intent> with Diagnosticable {
       }
     }
   }
+
+  /// Creates an overridable [Action] that allows itself to be overridden by the
+  /// closest ancestor [Action] in the given [context], if it exists.
+  ///
+  /// When invoked, the resulting [Action] tries to find the closest enabled
+  /// [Action] in [context] that handles the same type of [Intent] as this
+  /// [Action], then calls its [Action.invokeAsOverride] method. When no
+  /// enabled override [Action]s can be found, the resulting action has the same
+  /// behavior as this [Action].
```

Comment:
```suggestion
  /// enabled override [Action]s can be found, it defaults to this action.
```

I know you're trying to word it in this way because the implementation details mean that it's not actually returning `this` if it fails to find one, but I think that detail isn't really necessary to put in the docs: it just makes it harder to understand.